patches/tModLoader/Terraria/ID/TileID.TML.cs

```diff
@@ -69,6 +69,17 @@ partial class Sets
 			/// Tiles that are interpreted as a wall by nearby walls during framing, causing them to frame as if merging with this adjacent tile. Prevents wall from drawing within bounds for transparant tiles.
 			/// </summary>
 			public static bool[] WallsMergeWith = Factory.CreateBoolSet(Glass);
+
+			/// <summary>
+			/// New created sets to facilitate vanilla biome block counting including modded blocks. To replace the current hardcoded counts in SceneMetrics.cs
+			/// </summary>
+			public static bool[] CorruptBiome = Factory.CreateBoolSet(23, 24, 25, 32, 112, 163, 400, 398);
```

Comment:
Suggest, using `IntSet` instead, and we could put the 'score' values in here (1 normally, but it would support when a tile counts for multiple, or negative)
Eg `-10` evil for sunflowers